### ISSUE TITLE
Map coordinates

### ISSUE THREAD

**Issue by inputuserhere** [splewis/csgo-retakes#274]
Which folder are map coordinates read from? Also, is there any way to make throwing molotovs on T quicker?

csgo/addons/sourcemod/configs/retakes

csgo/serverfiles/csgo/configs/retakes

**Comment by Natanel-Shitrit** [comment]
### **"Which folder are map coordinates read from?"**
If you meant to ask for the spawns - it's in [`csgo/configs/retakes`](https://github.com/splewis/csgo-retakes/tree/master/configs/retakes) as you wrote.

### **Is there any way to make throwing molotovs on T quicker?**
You can change the speed of the grenades.
I don't know if it's allowed to link other plugins here, but you find it with a quick google search.

**Comment by inputuserhere** [comment]
@Natanel-Shitrit 

There are two folders with map coordinates. Not sure why. Also, the molotov throwing seems to be related to retakes, tried it on a different server and nothing. It's delayed for T, not CT.

**Comment by Natanel-Shitrit** [comment]
@inputuserhere 

[`csgo/addons/sourcemod/configs/retakes`](https://github.com/splewis/csgo-retakes/tree/master/cfg/sourcemod/retakes) has no map coordinates in it.

This config contains the convar values
(1) the plugin needs to run properly and
(2) other convars that change the overall gameplay that won't affect the plugin

**Comment by inputuserhere** [comment]
Would you happen to know the convar that removes the delay from T side to throw molly? Many of my players complain and I can't find it anywhere.

**Comment by Natanel-Shitrit** [comment]
Very unlikely, No cvar that i know of changes the molotov velocity.
This is probably caused by a sourcepawn plugin. (or a metamod extension)

**Comment by inputuserhere** [comment]
That's weird, I don't have anything other than retakes, sourcebans and sourcemod admin installed.

**Comment by borzaka** [comment]
Are you sure you installed retakes, and not executes? Because executes has nade delays. Show us your installed plugins list.

**Comment by inputuserhere** [comment]
Yes it's retakes_0.3.4. Are there any convars for the weapon allocator plugin? I don't see any on the main page.


 "Rock The Vote" (1.10.0.6484) by AlliedModders LLC
 "SourceBans++: SourceComms" (1.6.3) by Alex, SourceBans++ Dev Team
 "Fun Votes" (1.10.0.6484) by AlliedModders LLC
 "SourceBans++ Report Plugin" (1.6.3) by RumbleFrog, SourceBans++ Dev Team
 "SourceBans++: Main Plugin" (1.6.3) by SourceBans Development Team, SourceBans++ Dev Team
 "MapChooser" (1.10.0.6484) by AlliedModders LLC
 "Fun Commands" (1.10.0.6484) by AlliedModders LLC
 "Anti-Flood" (1.10.0.6484) by AlliedModders LLC
 "Nextmap" (1.10.0.6484) by AlliedModders LLC
 "Sound Commands" (1.10.0.6484) by AlliedModders LLC
 "SourceBans++: SourceSleuth" (1.6.3) by ecca, SourceBans++ Dev Team
 "[Retakes] Autoplant" (2.3.1) by B3none
 "Admin File Reader" (1.10.0.6484) by AlliedModders LLC
 "Basic Chat" (1.10.0.6484) by AlliedModders LLC
 "SourceBans++: Admin Config Loader" (1.6.3) by AlliedModders LLC, SourceBans++ Dev Team
 "CS:GO Retakes" (0.3.4) by splewis
 "Basic Commands" (1.10.0.6484) by AlliedModders LLC
 "Basic Votes" (1.10.0.6484) by AlliedModders LLC
 "SourceBans++: Bans Checker" (1.6.3) by psychonic, Ca$h Munny, SourceBans++ Dev Team
 "[Retakes] Bombsite HUD" (2.2.5) by B3none
 "CS:GO Retakes: site picker" (0.3.4) by splewis
 "Client Preferences" (1.10.0.6484) by AlliedModders LLC
 "Map Nominations" (1.10.0.6484) by AlliedModders LLC
 "Reserved Slots" (1.10.0.6484) by AlliedModders LLC
 "Admin Menu" (1.10.0.6484) by AlliedModders LLC
 "Basic Comm Control" (1.10.0.6484) by AlliedModders LLC
 "Player Commands" (1.10.0.6484) by AlliedModders LLC
 "Nikooo777" (1.0) by Nikooo777
 "Admin Help" (1.10.0.6484) by AlliedModders LLC
 "Basic Info Triggers" (1.10.0.6484) by AlliedModders LLC

**Comment by borzaka** [comment]
The weapon allocator plugin are optional and disabled by default. To enable it move them up from the `addons/sourcemod/plugins/disabled` directory to `addons/sourcemod/plugins`.

I still don't understand your problem; what nade delays? You need to show us some video what do you mean by that.

**Comment by Natanel-Shitrit** [comment]
The only plugin that seem suspicious is `"Nikooo777" (1.0) by Nikooo777`.
if it's not what causes the problem i recommend checking if everything is up to date, such as sourcemod it self.
